Tests/SQLKitTests/SQLKitTests.swift

```diff
@@ -678,4 +678,22 @@ CREATE TABLE `planets`(`id` BIGINT, `name` TEXT, `diameter` INTEGER, `galaxy_nam
             XCTFail("Could not decode row with keyDecodingStrategy \(error)")
         }
     }
+
+    func testUnion() throws {
+        try db.union(
+            db.select()
+                .column("id")
+                .from("t1")
+                .where("f1", .equal, "foo")
+                .limit(2),
+            db.select()
+                .column("id")
+                .from("t2")
+                .where("f2", .equal, "bar")
+                .limit(3)
+        ).run().wait()
+
+        XCTAssertEqual(db.results[0], "(SELECT `id` FROM `t1` WHERE `f1` = ? LIMIT 2) UNION (SELECT `id` FROM `t2` WHERE `f2` = ? LIMIT 3)")
```

Comment:
My local test with a Postgres db also checks that the binds are correctly enumerated, i.e. I can assert

```swift
        XCTAssertEqual(renderSQL(union.query),
                       #"(SELECT "id" FROM "t1" WHERE "f1" = $1 LIMIT 2) UNION (SELECT "id" FROM "t2" WHERE "f2" = $2 LIMIT 3)"#)
```

I don't think there's any way to test this with `TestDatabase` and maybe that's not an error that could occur. Just wanted to point it out!